DEBUG:root:Я на числе 0
DEBUG:root:Я на числе 1
DEBUG:root:Я на числе 2
DEBUG:root:Я на числе 3
DEBUG:root:Я на числе 4
DEBUG:root:Я на числе 5
DEBUG:root:Я на числе 6
DEBUG:root:Я на числе 7
DEBUG:root:Я на числе 8
DEBUG:root:Я на числе 9
DEBUG:root:Я на числе 10
DEBUG:root:Я на числе 11
DEBUG:root:Я на числе 12
DEBUG:root:Я на числе 13
DEBUG:root:Я на числе 14
DEBUG:root:Я на числе 15
DEBUG:root:Я на числе 16
DEBUG:root:Я на числе 17
DEBUG:root:Я на числе 18
DEBUG:root:Я на числе 19
DEBUG:root:Я на числе 20
DEBUG:root:Я на числе 21
DEBUG:root:Я на числе 22
DEBUG:root:Я на числе 23
DEBUG:root:Я на числе 24
DEBUG:root:Я на числе 25
DEBUG:root:Я на числе 26
DEBUG:root:Я на числе 27
DEBUG:root:Я на числе 28
DEBUG:root:Я на числе 29
DEBUG:root:Я на числе 30
DEBUG:root:Я на числе 31
DEBUG:root:Я на числе 32
DEBUG:root:Я на числе 33
DEBUG:root:Я на числе 34
DEBUG:root:Я на числе 35
DEBUG:root:Я на числе 36
DEBUG:root:Я на числе 37
DEBUG:root:Я на числе 38
DEBUG:root:Я на числе 39
DEBUG:root:Я на числе 40
DEBUG:root:Я на числе 41
DEBUG:root:Я на числе 42
DEBUG:root:Я на числе 43
DEBUG:root:Я на числе 44
DEBUG:root:Я на числе 45
DEBUG:root:Я на числе 46
DEBUG:root:Я на числе 47
DEBUG:root:Я на числе 48
DEBUG:root:Я на числе 49
DEBUG:root:Я на числе 50
DEBUG:root:Я на числе 51
DEBUG:root:Я на числе 52
DEBUG:root:Я на числе 53
ERROR:root:Ошибка:
Traceback (most recent call last):
  File "C:\Users\мвидео\PycharmProjects\my_workspace\Skillbox\Modul 2\logging\my_first_logging.py", line 12, in func_1
    x = "test" / 2
TypeError: unsupported operand type(s) for /: 'str' and 'int'
